doc/fileformat/cable_cell.rst

```diff
@@ -0,0 +1,440 @@
+
+.. label:: (region-def label:string reg:region)
+
+   This defines a ``label`` which can be used to refer to the region ``reg``.
+   For example:
+
+   .. code:: lisp
+
+      (region-def "my_region" (branch 1))
+
+   This expression identifies the branch with id 1 as "my_region".
+
+.. label:: (locset-def label:string ls:locset)
+
+   This defines a ``label`` which can be used to refer to the locset ``ls``.
+   For example:
+
+   .. code:: lisp
+
+      (locset-def "my_locset" (location 3 0.5))
+
+   This expression identifies the midpoint of the branch with id 3 as "my_locset".
+
+Any number of locset and region definitions can be grouped in a label dictionary as follows:
+
+.. label:: (label-dict [...def:region-def/locset-def])
+
+   This describes a label dictionary of zero or more region and locset definitons.
+   For example:
+
+   .. code:: lisp
+
+      (label-dict
+        (region-def "my_soma" (tag 1))
+        (locset-def "root" (root))
+        (region-def "all" (all))
+        (region-def "my_region" (radius-ge (region "my_soma") 1.5))
+        (locset-def "terminal" (terminal)))
+
+Decor
+-----
+
+The decor of a cable cell describes the dynamics and properties of the cell which can be assigned on
+:term:`regions <region>` or :term:`locsets <locset>`, or set as defaults on the entire cell.
+
+This table lists all supported dynamics and properties and whether they are *placeable* (i.e. they can
+be placed on one or more locations on the cell described by a locset); *paintable* (i.e. they can be set
+on an entire area of the cell described by a region) or *defaultable* (i.e. they are the default settings
+of the cell):
+
+.. csv-table:: Property applicability.
+   :widths: 20, 10, 10, 10
+
+                             ,         **placeable**, **paintable**, **defaultable**
+   initial membrane potential,         --,             ✓,             ✓
+   axial resistivity,                  --,             ✓,             ✓
+   temperature,                        --,             ✓,             ✓
+   membrane capacitance,               --,             ✓,             ✓
+   ion initial internal concentration, --,             ✓,             ✓
+   ion initial external concentration, --,             ✓,             ✓
+   ion initial reversal potential,     --,             ✓,             ✓
+   ion reversal potential method,      --,            --,             ✓
+   density mechanism,                  --,             ✓,            --
+   point mechanism,                    ✓,             --,            --
+   current clamp,                      ✓,             --,            --
+   threshold detector,                 ✓,             --,            --
+   gap junction site,                  ✓,             --,            --
+
+The various properties and dynamics of the decor are described as follows:
+
+.. label:: (membrane-potential val:real)
+
+   This describes an *initial membrane potential* object with value ``val`` (unit [mV]).
```

Comment:
Physicists nitpick: `[X]` means 'whatever the units of the quantity `X` are'. So, `[L] = m`, but `[m]` does not make sense in this context. Just drop the `[]`, saying 'unit' is enough. I know this is appearing in other place of Arbor.